src/data/services/EnterpriseSubsidyApiService.js

```diff
@@ -6,8 +6,20 @@ import { configuration } from '../../config';
 class SubsidyApiService {
   static baseUrl = `${configuration.ENTERPRISE_SUBSIDY_BASE_URL}/api/v1`;
 
+  static subsidyV2Url = `${configuration.ENTERPRISE_SUBSIDY_BASE_URL}/api/v2/subsidies`;
```

Comment:
nit: if we need to have both a `v1` and a `v2` constant, perhaps something like might work. I feel it'd be preferable if we could migrate the existing `v1` call to use the `v2` endpoint so we don't necessarily need to support both `v1` and `v2` together.

```js
class SubsidyApiService {
  static baseUrl = `${configuration.ENTERPRISE_SUBSIDY_BASE_URL}/api/`;
  static baseUrlV1 = `${baseUrl}/v1/`;
  static baseUrlV2 = `${baseUrl}/v2/`;
}
```

Reply:
yaaaaaaaa that's a good callout